pom.xml

```diff
@@ -67,118 +67,117 @@
       </exclusions>
     </dependency>
 
-    <!-- required plugins -->
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>jackson2-api</artifactId>
-      <version>2.7.3</version>
+      <version>2.9.7.1</version>
```

Comment:
Should align `com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.7.7` to `com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.9.7` (through kubernetes-client)